### PR TITLE
[#1541] Removing archived projects from the Xtext Community Website.

### PR DESCRIPTION
**Pull request by miklossy**
- Removing archived EMF IncQuery project.

Signed-off-by: Tamas Miklossy <miklossy@itemis.de>

### CODE REVIEW

**Comment by miklossy** [comment]
The [EMF IncQuery project](https://projects.eclipse.org/projects/modeling.incquery) has been archived and integrated into the [VIATRA project](https://www.eclipse.org/viatra/).

screenshot


Hence this entry is some kind of duplicate and can be removed from the [Xtext Community Website](https://www.eclipse.org/Xtext/community.html). 

@ujhelyiz Can you confirm that?

File: xtext-website/community.html [hunk 1]
```diff
@@ -373,14 +373,6 @@ <h2>What others have built with Xtext</h2>
 						<td>Cedric Vidal</td>
 					</tr>
 
-					<tr>
-						<td><a href="https://www.eclipse.org/incquery/">EMF&nbsp;IncQuery</a></td>
```
Comment (cdietrich) [inline xtext-website/community.html:377]: do we have an entry for viatra?

Comment (miklossy) [reply xtext-website/community.html:377]: Yes:
image

**Comment by miklossy** [comment]
Thanks for your quick reply @ujhelyiz ! If @cdietrich also agrees, I will merge this PR.

**Pull request merged**